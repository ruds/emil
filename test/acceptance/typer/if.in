if true then 1 else 2;
if 1 then 2 else 3; (* condition must be bool *)
if true then 1 else "hi"; (* branches must unify *)
val f = fn (a, b) => if a then b else "hi";
