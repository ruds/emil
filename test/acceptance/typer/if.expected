@0001:
val it : bigint
@0002: ERROR
@0003: ERROR
@0004:
val f : (bool * string) -> string
@0005:
